src/MetaModels/Attribute/Color/Color.php

```diff
@@ -110,12 +111,17 @@ public function sortIds($idList, $strDirection)
      *
      * @return array
      */
-    private function colorSort($colors)
+    private function colorSort($colors, $strDirection)
```

Comment:
Variable name should be only `$direction`, we are not using prefixes in new code.
Additionally, the parameter is not documented (as already indicated by the travis inspection).